corehq/apps/locations/models.py

```diff
@@ -281,15 +281,13 @@ def couch_locations(self, wrapped=True):
         return locations
 
     def accessible_to_user(self, domain, user):
-        from corehq.apps.locations.util import get_locations_and_children
-
         if user.has_permission(domain, 'access_all_locations'):
             return self.all()
 
         assigned_location_ids = user.get_location_ids(domain)
         if not assigned_location_ids:
-            return self.none()  # No locations are accessible to this user
-        return self.all() & get_locations_and_children(assigned_location_ids)
+            return self.none()  # No locations are assigned to this user
+        return self.all() & SQLLocation.active_objects.get_locations_and_children(assigned_location_ids)
```

Comment:
@esoergel Changing this to object. Though i don't see why we won't follow the same principle as [here](https://github.com/dimagi/commcare-hq/pull/13362#discussion-diff-90930237R353) to call for the calling manager and make this `self.get_locations_and_children`